notebook/static/base/js/i18nload.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Plugin to load a single locale.
+ */
+define([
+	"require",
+	"module",
+	 // These are only here so that the optimizer knows which ones we MIGHT load.
+	 // We will actually only load the ones we need. There should be one entry
+	 // here for each language you want to support.
+	 // For example, for German....
+	 // "json!base/../../i18n/de/LC_MESSAGES/nbjs.json"

```

Comment:
Does the optimiser look at commented values, or is this comment describing something that happens if it's uncommented? If we need uncommented values here, what stops them all being loaded when `i18nload` is loaded?